Assets/XML/Text/Victory.xml

```diff
@@ -5094,7 +5094,7 @@
 		<Tag>TXT_KEY_VICTORY_TRANSSIBERIAN_RAILWAY</Tag>
 		<English>Railroad from Moscow to a Siberian port</English>
 		<French>Chemin de fer de Moscou &#224; un port sib&#233;rien</French>
-		<German>Railroad from Moscow to a Siberian port</German>
+		<German>Eisenbahn von Moskau an einen sibirischen Hafen</German>
```

Comment:
zu einem

Reply:
Grammar Nazis get the bullet too.

Reply:
Grammar is fine, I just want a different preposition.